lib/_emerge/JobStatusDisplay.py

```diff
@@ -258,14 +259,16 @@ def _display_status(self):
             f.push_style(number_style)
             f.add_literal_data(running_str)
             f.pop_style()
-            f.add_literal_data(" running")
+            f.add_literal_data(" running ")
+            f.add_literal_data(f"{self._pkg} ")
 
         if self.failed:
             f.add_literal_data(", ")
             f.push_style(number_style)
             f.add_literal_data(failed_str)
             f.pop_style()
-            f.add_literal_data(" failed")
+            f.add_literal_data(" failed ")
+            f.add_literal_data(f"{self._pkg} ")
```

Comment:
Aren't you displaying it twice? Also, you probably need to shorten the other messages to make it all fit on 80-char terminal.